src/tmx/TmxUtils/test/J2735MessageTest.cpp

```diff
@@ -763,9 +774,19 @@ TEST_F(J2735MessageTest, EncodeTravelerInformation){
 	#if SAEJ2735_SPEC < 2020
 	string expectedHex = "001f526011c35d000000000023667bac0407299b9ef9e7a9b9408230dfffe4386ba00078005a53373df3cf5372810461b90ffff53373df3cf53728104618129800010704a04c7d7976ca3501872e1bb66ad19b2620";
 	#else
-	string expectedHex = "001f6820100000000000f9e1d8d0803299b9eac27a9baa74232000000fcec0a9df4028007e53373d584f53754e846400b720000000b8f5374e3666e7ac5013ece3d4ddc1099b9e988050538f5378f9666e7a5a814140034000dea1f5e5db2a083a32e1c80a048b26a22100";
+	string expectedHex = "001f45201000000000019b3915c0807299b9ea847a9b9dea2001fffe4fd0f23a000078005253373d508f5373bd4400325fffe05b94cdcf53e3d4dcef30e7aa58000002250e1a8880";
 	#endif
-	ASSERT_EQ(expectedHex, timEnc.get_payload_str());			
+	ASSERT_EQ(expectedHex, timEnc.get_payload_str());
```

Comment:
NIT: For unit tests use 
```cpp
EXPECT_EQ(...)
// instead of 
ASSERT_EQ(...)
```

EXPECT_EQ:
Generates a non-fatal failure if the values are not equal.
The test continues to execute even after the failure, allowing you to potentially discover multiple failures within a single test case.
ASSERT_EQ:
Generates a fatal failure if the values are not equal.
The test execution aborts immediately, exiting the current function.

Reply:
In most cases we want EXPECT_EQ. This allows us to see all the errors at once. ASSERT is normally used when you want execution to stop because some assertion has failed and makes all other down stream code invalid.